Cargo.toml

```
@@ -20,10 +20,10 @@ include = [
 ]
 
 [dependencies]
-bitflags = "1.0"
-crc32fast = "1.2.0"
-deflate = "1.0"
-miniz_oxide = "0.5.1"
+bitflags = "^1.0"
+crc32fast = "^1.2.0"
+flate2 = "^1.0"
+miniz_oxide = "^0.5.0"
```

There's no need to add the `^` symbol here. Cargo treats `"1.0"` the same as `"^1.0"`

---

I guess I've been doing too much npm recently. Cargo docs are actually pretty clear about it (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements)

```suggestion
bitflags = "1.0"
crc32fast = "1.2.0"
flate2 = "1.0"
miniz_oxide = "0.5.0"
```